.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,19 @@
+# Copyright The Ratify Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Template created based on verbiage from https://github.com/notaryproject/notation-go/blob/v1.2.1/.github/ISSUE_TEMPLATE/config.yml
```

Comment:
It should be based on `notation` instead of `notation-go`, shouldn't it?

Reply:
Updated.